From: Philip Withnall <pwithnall@endlessos.org>
Date: Mon, 9 Jan 2023 12:20:10 +0000
Subject: glib-compile-schemas: Fix typo in comparison function

This caused schemas and override files to be sorted incorrectly.

Fixes a regression introduced in commit 861e82efbc.

Spotted by John Wudrick in
https://gitlab.gnome.org/GNOME/glib/-/merge_requests/3155#note_1635044.

Signed-off-by: Philip Withnall <pwithnall@endlessos.org>
Origin: upstream, 2.75.3, commit:9d5bcc94919b7502d41de4d1de3630f47d4abb09
---
 gio/glib-compile-schemas.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gio/glib-compile-schemas.c b/gio/glib-compile-schemas.c
index 2352f22..acabfb6 100644
--- a/gio/glib-compile-schemas.c
+++ b/gio/glib-compile-schemas.c
@@ -1870,7 +1870,7 @@ compare_strings (gconstpointer a,
                  gconstpointer b)
 {
   const gchar *one = a;
-  const gchar *two = a;
+  const gchar *two = b;
   gint cmp;
 
   cmp = g_str_has_suffix (two, ".enums.xml") -
